tracks/ruby/exercises/resistor-color-duo/mentoring.md

```diff
@@ -1,4 +1,4 @@
-Resistor Color Duo is a side exercise, unlocked by Hello World
+Resistor Color Duo is a side exercise, unlocked by Hello World (meaning this _may_ be the first time they get mentoring!). 
 
 ### New Concepts
 Array, Array#index, `map`, Constant, chaining methods
```

Comment:
```suggestion
`Array`, `Array#index`, `map`, Constant, chaining methods
```

Reply:
I don't agree, but I don't want to block the merging over this.

Reply:
:)